pkg/sql/alter_database.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/sql/sqlerrors"
 	"github.com/cockroachdb/cockroach/pkg/sql/sqltelemetry"
+	"github.com/cockroachdb/cockroach/pkg/sql/username"
```

Comment:
this import confused me when i reviewed this commit but got lost somewhere. i don't know if you want to fix it.

Reply:
I fixed this.